internal/git/git.go

```diff
@@ -183,40 +185,49 @@ func (c Client) FindExtinctions(projKey string, flags []string, matcher search.M
 		if err != nil {
 			return nil, err
 		}
-		patch, err := changes.Patch()
+		patch, err := changes.PatchContext(context.Background())
```

Comment:
This allows us to access `FilePatches` below. So now we can loop over the files in the commit to see if they're in the subdirectory for that matcher.